src/index.css

```diff
@@ -31,3 +31,15 @@ code {
 .align-center {
   text-align: center;
 }
+
+.navbar a{
+  text-decoration: none;
+  color: skyblue;
+  font-size: 18px;;
+  padding-right: 20px;
+}
+.navbar{
```

Comment:
```suggestion

.navbar {
```

Reply:
What is the issue, if we do not add a space?